README.md

```diff
@@ -30,50 +29,47 @@ $ pip install meeshkan
 
 Note that `meeshkan` requires **Python 3.6+.**
 
-## Python API
+## Hello world
 
-### Quick start
+The basic Meeshkan flow is **collect, build and mock.**
+1. To start, **collect** data from recorded server traffic and/or OpenAPI specs.
+1. Then, **build** a schema that unifies these various data sources.
+1. Finally, use this schema to create a **mock** server of an API.
 
-```python
-from meeshkan.schemabuilder.update_mode import UpdateMode
-from meeshkan.schemabuilder import build_schema_batch, build_schema_online, update_openapi
-from openapi_typed_2 import OpenAPIObject
-import meeshkan
-from typing import List
-import json
-from http_types import HttpExchange, Request, Response, ResponseBuilder, RequestBuilder, HttpMethod
-
-
-def read_http_exchanges() -> List[HttpExchange]:
-    """Read HTTP exchanges from the source of your choice.
-    """
-    request: Request = RequestBuilder.from_url(
-        "https://example.org/v1", method=HttpMethod.GET, headers={})
+```bash
+$ meeshkan record -r --daemon # start the recorder in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # record traffic
+$ meeshkan record --stop-daemon # stop the daemon
+$ meeshkan build
+$ meeshkan mock -r --daemon # start the mocking server in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # mock traffic
+$ meeshkan mock --stop-daemon # stop the daemon
+```
 
-    response: Response = ResponseBuilder.from_dict(dict(
-        statusCode=200, body=json.dumps({"hello": "world"}), headers={}))
+## Collect
 
-    http_exchange: HttpExchange = HttpExchange(request=request, response=response)
+To build a Meeshkan spec, one must first **collect** recordings of server traffic and/or OpenAPI server specs.
 
-    return [http_exchange]
+To record API traffic, Meeshkan provides a recorder that can capture API traffic using a proxy.
 
+```bash
+$ pip install meeshkan # if not installed yet
+$ meeshkan record
+```
 
-http_exchanges = read_http_exchanges()
+This starts Meeshkan as a reverse proxy on the default port of `8000`.  For example, with curl, you can use Meeshkan as a proxy like so.
 
-# Build OpenAPI schema from a list of recordings
-openapi: OpenAPIObject = build_schema_batch(http_exchanges)
+```bash
+$ curl http://localhost:8000/http://api.example.com
+```
 
-# Build schema from an iterator
-openapi: OpenAPIObject = build_schema_online(iter(http_exchanges), UpdateMode.GEN)
+By default, the recording proxy treats the path as the target URL and writes a `.jsonl` file containing logs of all server traffic to a `logs` directory.  All logs are created in the [`http-types`](https://github.com/meeshkan/http-types) format.
```

Comment:
Should we mention that Meeshkan always expects recordings to be in the `http-types` format?

Reply:
Also what JSON Lines is? (similar to how we link to it/explain it in the `Build` section)